python/pyarrow/parquet/core.py

```diff
@@ -1163,13 +1163,13 @@ def _get_pandas_index_columns(keyvalues):
 buffer_size : int, default 0
     If positive, perform read buffering when deserializing individual
     column chunks. Otherwise IO calls are unbuffered.
-partitioning : pyarrow.dataset.Partitioning or str or list of str, \
+partitioning : :py:class:`pyarrow.dataset.Partitioning` or str or list of str, \
```

Comment:
> I thought this shouldn't be needed, because sphinx was set up to do this manually?

I'm curious what config you refer to here. 
I found the [default_role](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role) feature, but that at least requires single back ticks around the reference